lib/src/multi_channel.dart

```diff
@@ -132,15 +132,15 @@ class _MultiChannel<T> extends StreamChannelMixin<T>
   /// it's coming from a channel that was originally created locally.
   var _nextId = 1;
 
-  _MultiChannel(this._inner) {
+  _MultiChannel(StreamChannel<dynamic> inner) : _inner = inner {
```

Comment:
hah, that's an interesting one

Reply:
Yeah, it's unusual to have a field which always starts with a value then eventually becomes null.